packages/node/src/node.ts

```diff
@@ -226,8 +229,8 @@ export class Node {
    *     solely to the deffered promise's resolve callback.
    */
   private async handleReceivedMessage(msg: NodeMessage) {
-    if (!NODE_EVENTS[msg.event]) {
-      console.error(`Received message with unknown event type`);
+    if (!Object.values(NODE_EVENTS).includes(msg.event)) {
```

Comment:
I hate that TypeScript forces the use of Object.values for such a trivial task :(OI hat

Reply:
Yeah I noticed with the other change it was always evaluating to `true` so have to revert

Reply:
Yeah, I re-read it and sorry 'bout that, I was talking non-sense, lol.